### PR TITLE
Amend dynamic volume list doc

### PR DESCRIPTION
**Pull request by tengqm**
The dynamic volume list feature has been documented but the feature gate
related was not there yet.

Closes: #9105

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ different Kubernetes components.
 | `APIListChunking` | `true` | Beta | 1.9 | |
 | `APIResponseCompression` | `false` | Alpha | 1.7 | |
 | `AppArmor` | `true` | Beta | 1.4 | |
+| `AttachVolumeLimit` | `true` | Alpha | 1.11 | |
```
Comment (stewart-yu) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md]: default value is `false`

Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Good catch, sir!

**Comment by mistyhacks** [comment]
I'm going to close and reopen this to retrigger the flakey test.

**Pull request merged**